### ISSUE TITLE
Quick view area is too small under certain circumstances

### ISSUE THREAD

**Issue by afillatre** [la-team/light-admin#231]
In the list view, when clicking on the little arrow to display the entity's quick view, the quick view area is no higher than the height of the other entities in the list view.
In other words, when there is only 1 entity we want to see the quick view, clicking on the arrow seems to do nothing because the height of the table area is too small to display it. We need to scroll in this area to see the result. When there are 2 entities and we want to display the quick view of the first, we only see the first entity row, and so forth.

**Comment by ikostenko** [comment]
reproducible in 1.2.0.RC1

**Comment by afillatre** [comment]
Yes, sorry, I forgot to add the version. I'm using this one as well.

**Comment by lincetto** [comment]
I have a different problem with the quick view in the 1.2.0.BUILD-SNAPSHOT version: when I click the arrow, the quick view appears for a while and then disappears (and the table is reloaded).

**Comment by ikostenko** [comment]
@lincetto 
Which browser and operating system are you using?

**Comment by max-dev** [comment]
Reproduced & investigating currently. [Regression defect]
